Removed UsePersistentCookiesForExplorerView requirement (#3159)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/sharepoint-view-in-edge.md

        > [!NOTE]
        > Ensure the management shell version is 16.0.21610.12000 or higher or the ViewInFileExplorerEnabled option will not be available.

> 7. **(Required)** "View in File Explorer" requires persisted cookies to operate correctly, user need to select "Yes" when prompted for "Stay signed in?" at the time of sign-in.

Reviewer: ```suggestion
7. **(Required)** _View in File Explorer_ requires persisted cookies to operate correctly; you need to select **Yes** when prompted to **Stay signed in?** when you sign in.
```
